src/index.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import "./index.css";
+import App from "./App";
+import reportWebVitals from "./reportWebVitals";
```

Comment:
Feel free to remove `reportWebVitals` to keep this tidy

Reply:
removed